### PR TITLE
Encode url

### PR DESCRIPTION
**Pull request by MaxMernikov**
encoding request image url to thumbor for save params
used only for safe requests only

fix issues like as:
https://github.com/thumbor/thumbor/issues/813
https://github.com/thumbor/thumbor/issues/814

### CODE REVIEW

**Comment by rafaelcaricio** [comment]
@MaxMernikov Thank you for the PR. Would you mind to write a test case for that?

**Comment by rafaelcaricio** [comment]
We need also to update the tests in general.

File: lib/thumbor_rails/helpers.rb [hunk 1]
```diff
@@ -8,7 +8,11 @@ def thumbor_url(image_url, options = {})
       if ThumborRails.force_no_protocol_in_source_url
         image_url = image_url.sub(/^http(s|):\/\//, '')
       end
-
+      
+      unless options[:unsafe]
+        image_url = ERB::Util.url_encode(image_url)
```
Comment (masom) [inline lib/thumbor_rails/helpers.rb]: Generally speaking the image url should always be encoded :)

**Comment by rafaelcaricio** [comment]
Would be nice to have the tests updated. So we can merge and release a new version. I will try to do it when I get some time. But feel free to update this PR or let me know if you cannot do it now.

File: lib/thumbor_rails/helpers.rb (hunk 1 above)
Comment (MaxMernikov) [inline lib/thumbor_rails/helpers.rb]: It's for more comfortable using on browser code inspector (for change size or filter). Encoded symbols like & ? (%26%20%3F) poor for editing.

**Comment by MaxMernikov** [comment]
Perhaps I can make it this weekend.

**Comment by MaxMernikov** [comment]
@rafaelcaricio need help 
https://travis-ci.org/rafaelcaricio/thumbor_rails/jobs/241587149

**Comment by rafaelcaricio** [comment]
I think we should drop support for very old versions of Ruby/Rails. I opened a PR for that https://github.com/rafaelcaricio/thumbor_rails/pull/11. Please scream if you see anything wrong.

**Comment by rafaelcaricio** [comment]
@MaxMernikov Please rebase with master, and the problem you are having should be fixed.

**Comment by rafaelcaricio** [comment]
@MaxMernikov let me know if you can do the rebase.

**Comment by MaxMernikov** [comment]
slowly but surely
I hope

**Comment by rafaelcaricio** [comment]
@MaxMernikov Thank you for the PR.

**Comment by pashazel** [comment]
@MaxMernikov, @rafaelcaricio nice work!

**Pull request merged**